hooks/post-checkout

```diff
@@ -2,12 +2,12 @@
 
 set -euo pipefail
 
-function trigger() {
+trigger() {
   local pr="$1"
   local branch="$2"
```

Comment:
@zsims `branch` looks like an unused variable—or am I missing something?

Reply:
You're right, this should probably be set to `${BUILDKITE_BRANCH}` at the caller -- or just removed. Are you happy to add that to this PR? If not, no stress I can address it after this lands